app/controllers/application_controller.rb

```diff
@@ -33,7 +33,7 @@ def not_found
     ).attributes, status: :not_found
   end
 
-  private
+private
```

Comment:
same as the editor?

Reply:
Done on https://github.com/ministryofjustice/fb-metadata-api/pull/172/commits/5fbe5d8d309b5a02a3ffe7f4514efeefda93728a